samples/subsys/fs/fs_sample/sample.yaml

```diff
@@ -31,6 +31,18 @@ tests:
   sample.filesystem.fat_fs.nrf52840dk_nrf52840:
     build_only: true
     platform_allow: nrf52840dk/nrf52840
+  sample.filesystem.fat_fs.nrf52840dk_nrf52840_ram_disk:
+    build_only: true
```

Comment:
Why build only? (other than that the other testcases already have this set)

Reply:
I am not sure why we want to run this every time, but we want to know whether it still builds.

Reply:
Fair enough- I guess the runtime overhead is a concern for testing teams?

Reply:
Yeah, in reality the run should be done by test and verified - that way we will know whether it works internally.
The sample should be just like "Hello world" where we assume that everything works and just show people, in very simplified form, how to do things.
I was asked how do the disk in RAM, several times, and now I can just point to sample.
Building is still taking some CI time, so not perfect anyway.